src/dfcx_scrapi/core/agents.py

```diff
@@ -358,6 +367,7 @@ def export_agent(self, agent_id: str, gcs_bucket_uri: str) -> str:
         request = types.agent.ExportAgentRequest()
         request.name = agent_id
         request.agent_uri = gcs_bucket_uri
+        request.environment = environment
```

Comment:
@Omerside @MRyderOC 

While technically this does work when using the `Environment ID`, we should really implement a better UX here by allowing the user to provide the `Environment Display Name` and we perform the ID lookup internally. 

It's much more likely that a user will know they want to export "PROD" or "MyCustomEnvironment" vs. the long Env ID string.
Also, providing the `environment_id` requires the user to import 2 classes (Agents and Environments) and implement additional code that we could just implement for them.

Here's my suggestions:
- Implement a `get_environments_map` in `environments.py` similar to the other `get_x_map` functions around the library (See Intents, Flows, Pages, etc. for inspiration)
- Import the Environments class into the `agents.py` file
- In `export_agent`, implement the map lookup ONLY if the user provides the `environment` arg (otherwise it's a wasted API call)
- Also, only add `request.environment` if the user provides `environment` as an arg, otherwise don't modify